package.json

```diff
@@ -30,19 +30,19 @@
     "@types/chai": "^4.2.15",
     "@types/fast-levenshtein": "^0.0.1",
     "@types/lodash": "^4.14.166",
-    "@types/mocha": "^8.2.0",
-    "@types/node": "^14.14.31",
-    "@types/supports-color": "^7.2.0",
-    "chai": "^4.2.0",
+    "@types/mocha": "^8.3.2",
+    "@types/node": "^14.14.37",
+    "@types/supports-color": "^7.2.1",
+    "chai": "^4.3.4",
     "eslint": "^7.21.0",
     "eslint-config-oclif": "^3.1.0",
     "eslint-config-oclif-typescript": "^0.2.0",
-    "mocha": "^8.3.0",
+    "mocha": "^8.3.2",
     "ts-node": "^9.1.1",
-    "typescript": "3.8.3"
+    "typescript": "^4.2.4"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=14.0.0"
```

Comment:
I'm starting to second guess this. Since node 12 is still a LTS until mid-2022, I wonder if it is a good move to require 14. I guess another way to ask this, is it ok the unified CLI requires node 14? Maybe that is... we can also see what kind of reaction we get when we do the announcement post.